scripts/create-new-package.bash

```diff
@@ -59,6 +59,7 @@ case "$META" in
    fi
    if [[ ! -d $META_NAME ]]; then
      echo "ERROR: metapackage with the name '$META_NAME' does not exist! Exiting..."
```

Comment:
We should here use global functions 'print error and exit' from RTW defines file